src/world/components/lib/getRandomTransition.js

```diff
@@ -0,0 +1,7 @@
+const matchTransitionFor = require('./matchTransitionFor');
+
+module.exports = function getRandomTransition(world) {
+  const moveableSet = world.things.filter(thing => thing.locations.length > 1);
+  const randomThing = moveableSet[Math.floor(Math.random() * moveableSet.length)];
+  return randomThing && [matchTransitionFor(randomThing, world.numRules, world.rules), randomThing];
+}
```

Comment:
Can you provide a brief overview of the intention of this method so that I can document it (and better understand it myself)?